pkg/controller/v1beta1/inferenceservice/controller_test.go

```diff
@@ -1362,7 +1363,7 @@ var _ = Describe("v1beta1 inference service controller", func() {
 		})
 	})
 
-	Context("When creating an inference service with a ServingRuntime which is disabled", func() {
+	Context("When creating an inference service with a ServingRuntime which is disabled", func(){
```

Comment:
formatting issue?

Reply:
Sorry, this was our mistake. We will make the necessary corrections.